src/generate_batch.py

```diff
@@ -32,23 +32,29 @@ def generate_blink_seq(num_frames):
             frame_id = frame_id+start+9
         else:
             break
-    return ratio 
+    return ratio
 
-def generate_blink_seq_randomly(num_frames):
+def generate_blink_seq_randomly(num_frames, idlemode=False):
     ratio = np.zeros((num_frames,1))
-    if num_frames<=20:
+    min_start = 20 if idlemode else 10
+    if num_frames <= min_start * 2:
         return ratio
     frame_id = 0
     while frame_id in range(num_frames):
-        start = random.choice(range(min(10,num_frames), min(int(num_frames/2), 70))) 
```

Comment:
@smetak13  Nejsem si jistý, zda toto nějak neupravit. Člověk mrká cca 12x za minutu - to je cca 1x za 5s. Ale asi to může být range mezi 4-6s. Ale nemělo by se stát, že mrkne 3x rychle posobě.

Reply:
@daviddudas99 Mrkání upraveno na interval 2-4 vteřiny a zkráceno z 5 framů na 3